src/Symfony/Component/Ldap/Adapter/EntryManagerInterface.php

```diff
@@ -40,4 +40,13 @@ public function update(Entry $entry);
      * @param Entry $entry
      */
     public function remove(Entry $entry);
+
+    /**
+     * Renames an entry on the Ldap server.
+     *
+     * @param Entry  $entry
+     * @param string $newRdn
+     * @param bool   $removeOldRdn
+     */
+    public function rename(Entry $entry, $newRdn, $removeOldRdn = true);
```

Comment:
Hum, this interface should still actually be marked as private, and was only added in 3.2.
Write support is only in its infancy, and is not completely mature yet, so we may need to add new methods at later points (`move(Entry $entry, $newRoot)` for example).

We could create another interface which extends the EntryManagerInterface or a separate interface for newer methods, which could be merged in the main interface for Symfony 4.0 (See what was done for `LdapClientInterface` and `LdapInterface`, for example).

On the other hand, the query part is stable, and should not change in the upcoming months.